<?xml version="1.0"?>
<launch>

    <!-- RVIZ -->
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find gps_path_pub)/launch/gpstopdown.rviz" output="screen"/>

    <!-- Path publisher -->
    <node name="gps_path_pub" pkg="gps_path_pub" type="gps_path_pub" output="screen">

        <param name="save_file" type="bool"   value="true" />
        <param name="file_path" type="str"    value="/home/patrick/workspace/catkin_ws_init/src/open_vins/ov_data/rpng_ironsides/neighborhood_02.txt" />

        <remap from="/mti0/sensor/fix_navsat" to="/reach02/tcpfix" />

    </node>

</launch>